wdl2cwl/main.py

```diff
@@ -217,8 +218,21 @@ def main(argv: List[str]) -> str:
     # ^ converts multine line strings to nice multiline YAML
     yaml.dump(cwl_result, result_stream)
     yaml.dump(cwl_result, sys.stdout)
+
     return result_stream.getvalue()
 
 
+def main() -> None:
+    """Command-line parsing."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("workflow", help="WDL workflow")
+    parser.add_argument("-d", "--directory", help="Directory to store CWL files")
+    args = parser.parse_args()
```

Comment:
Consider writing a test that using monkeypatch to temporarily set `sys.argv` to a certain value: https://docs.pytest.org/en/6.2.x/monkeypatch.html#monkeypatching-returned-objects-building-mock-classes